nix/inferno-ml/migrations/v1-create-tables.sql

```diff
@@ -66,7 +66,11 @@ create table if not exists params
   ( id serial primary key
     -- Script hash from `inferno-vc`
   , script bytea not null references scripts (id)
-  , model integer references mversions (id)
+    -- Array of model version IDs; unfortunately we lose referential
+    -- integrity because Postgres does not allow arrays of foreign keys.
+    -- However, we don't allow model version deletion anyway, which
+    -- mitigates this to some degree
+  , models integer[] not null
```

Comment:
I see now why. So this is a many-to-many relationship, right? To regain referential integrity this could be modeled as
```sql
create table model_param (
  model integer references mversions (id) on update cascade on delete cascade,
  param integer references params (id) on update cascade on delete cascade
)
```
And queries adjusted accordingly using `array_agg` to flatten the left join between `param` and `model_param` back into an array